### ISSUE TITLE
E: Version '2.3.2+dfsg-1~exp2.pgdg80+1' for 'postgresql-9.6-postgis-2.3' was not found

### ISSUE THREAD

**Issue by narnikgamarnikus** [appropriate/docker-postgis#60]
Hey. When I try to use it with Django, I get such a mistake.

Previously, everything worked, but now this error:

```
E: Version '2.3.2+dfsg-1~exp2.pgdg80+1' for 'postgresql-9.6-postgis-2.3' was not found
E: Version '2.3.2+dfsg-1~exp2.pgdg80+1' for 'postgresql-9.6-postgis-2.3-scripts' was not found
E: Version '2.3.2+dfsg-1~exp2.pgdg80+1' for 'postgis' was not found

```

**Comment by cecemel** [comment]
It seems to work to change it to: POSTGIS_VERSION 2.3.3+dfsg-1.pgdg80+1

**Comment by tdamsma** [comment]
@md5 done, see https://github.com/appropriate/docker-postgis/pull/63

**Comment by md5** [comment]
@cactail That's addressed in #64 by removing the version constraint on the `postgis` package.

**Comment by mhyousefi** [comment]
I'm doing the following:
```
FROM postgres:10
MAINTAINER Mike Dillon <mike@appropriate.io>
ENV POSTGIS_MAJOR 2.4
ENV POSTGIS_VERSION 2.3.3+dfsg-1.pgdg80+1
RUN apt-get update \
      && apt-get install -y --no-install-recommends \
           postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR=$POSTGIS_VERSION \
           postgresql-$PG_MAJOR-postgis-$POSTGIS_MAJOR-scripts=$POSTGIS_VERSION \
           postgis=$POSTGIS_VERSION \
      && rm -rf /var/lib/apt/lists/*
RUN mkdir -p /docker-entrypoint-initdb.d
COPY ./initdb-postgis.sh /docker-entrypoint-initdb.d/postgis.sh
COPY ./update-postgis.sh /usr/local/bin
```

And I still get:
```
E: Version '2.3.3+dfsg-1.pgdg80+1' for 'postgresql-10-postgis-2.4' was not found
E: Version '2.3.3+dfsg-1.pgdg80+1' for 'postgresql-10-postgis-2.4-scripts' was not found
E: Version '2.3.3+dfsg-1.pgdg80+1' for 'postgis' was not found
```
